operate/qa/pom.xml

```diff
@@ -24,7 +24,7 @@
   <properties>
     <!-- properties for ImportSeveralVersionsTest and import-old-zeebe-tests module -->
     <version.zeebe.old>8.1.0</version.zeebe.old>
-    <version.zeebe.docker.current>8.6.0-alpha5</version.zeebe.docker.current>
+    <version.zeebe.docker.current>SNAPSHOT</version.zeebe.docker.current>
```

Comment:
For posterity: We clarified this. It was an issue before as this would then test the importer 8.7. But this PR also changes the infrastructure to make use of the camunda exporter then it is not an issue.